bindings/python/tools/python/ostk/astrodynamics/pytrajectory/pystate.py

```
@@ -108,8 +124,21 @@
         CartesianVelocity.default(),
     ]
 
-    if all(column in data for column in eci_columns):
-        frame = Frame.GCRF()
+    match_groups: list[re.Match] = [
+        re.match(CANONICAL_FORMAT, column) for column in data.keys()
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)